ensime_shared/spec/features/suggestion_format.feature

```diff
@@ -13,22 +13,6 @@ Feature: Format a Completion Suggestion
     | <byname>[ByNameType]   | => ByNameType |
     | <repeated>[VarargType] | VarargType*   |
 
-  Scenario: Format single type parameter
-    Given A list of type parameters:
-      | tparam |
-      | A      |
-    When We concat the type parameters
-    Then We get the format [A]
-
-  Scenario: Format multiple type parameters
-    Given A list of type parameters:
-      | tparam |
-      | A      |
-      | B      |
-      | C      |
-    When We concat the type parameters
-    Then We get the format [A, B, C]
```

Comment:
I had looked back at #273 and seen that, but I didn't completely understand what you meant on the ensime-server ticket about what was missing. Now I do, after looking at the server responses more closely in that situation, and I agree that type params should be added to the response (and would add minimally to payloads).

As of this moment on this branch, it's dead code and I prefer to remove dead code as soon as it's dead, that's what version control is for 😁  I should have mentioned you or commented on #276, but now you're on top of it already 😄